src/renderer/page/file/view.jsx

```diff
@@ -160,99 +145,79 @@ class FilePage extends React.Component<Props> {
     }
 
     return (
-      <Page extraPadding>
-        {!claim || !metadata ? (
-          <section>
-            <span className="empty">{__('Empty claim or metadata info.')}</span>
-          </section>
-        ) : (
-          <section className="card">
-            {showFile && (
-              <FileViewer className="content__embedded" uri={uri} mediaType={mediaType} />
-            )}
-            {!showFile &&
-              (thumbnail ? (
-                <Thumbnail shouldObscure={shouldObscureThumbnail} src={thumbnail} />
-              ) : (
-                <div
-                  className={classnames('content__empty', {
-                    'content__empty--nsfw': shouldObscureThumbnail,
-                  })}
-                >
-                  <div className="card__media-text">
-                    {__("Sorry, looks like we can't preview this file.")}
-                  </div>
-                </div>
-              ))}
-            <div className="card__content">
-              <div className="card__title-identity--file">
-                <h1 className="card__title card__title--file">{title}</h1>
-                <div className="card__title-identity-icons">
-                  {isRewardContent && (
-                    <Icon iconColor="red" tooltip="bottom" icon={icons.FEATURED} />
-                  )}
-                  <FilePrice uri={normalizeURI(uri)} />
+      <Page forContent>
+        <section className="content__wrapper">
+          {showFile && <FileViewer className="content__embedded" uri={uri} mediaType={mediaType} />}
+          {!showFile &&
+            (thumbnail ? (
+              <Thumbnail shouldObscure={shouldObscureThumbnail} src={thumbnail} />
+            ) : (
+              <div
+                className={classnames('content__empty', {
+                  'content__empty--nsfw': shouldObscureThumbnail,
+                })}
+              >
+                <div className="card__media-text">
+                  {__("Sorry, looks like we can't preview this file.")}
```

Comment:
Single quotes for consistency

Reply:
Prettier automatically will change single quotes with a `\'` to double quotes and a single quote for the apostrophe.